tools/generate_tables_sql_and_testNames.php

```diff
@@ -29,6 +29,7 @@
 require_once "../php/libraries/Database.class.inc";
 require_once "../php/libraries/NDB_Config.class.inc";
 require_once "../php/libraries/NDB_BVL_Instrument.class.inc";
+require_once "./generic_includes.php";
```

Comment:
The above 3 libraries are included by `generic_includes` so no need to include them anymore.